docs/components/modeler/desktop-modeler/process-applications.md

```diff
@@ -12,13 +12,19 @@ import StartInstanceImg from './img/process-applications/start-instance.png'
 
 Desktop modeler recognizes [process applications](../../concepts/process-applications.md) you build and offers you advanced editor intelligence, deployment, and execution features within the context of such an application. To identify the boundaries of a process application, Desktop Modeler searches for a `.process-application` file in the root of your project.
 
-For instance, a consumer loan approval process application might contain:
+In professional software development, a typical process application contains resources such as BPMN, DMN, and Form files. These live alonside [job workers](/components/concepts/job-workers.md), implementing process logic, additional application code, and tests. How exactly your project is structured may vary depending on the implementation language, libraries, and frameworks you use.
```

Comment:
[❓] Do we want to mention RPA scripts since they are supported by desktop modeler ? Since it might now fall under a typical process app and it is not part of the example I believe it is ok not to mention them.

Reply:
> Since it might now fall under a typical process app and it is not part of the example I believe it is ok not to mention them.

Yes, we don't have to mention everything. I think `such as` indicates that this is an open list.